io.openliberty.java.internal_fat_misc/java17/README.md

```diff
@@ -0,0 +1,93 @@
+# Repo for quick testing of Liberty apps
+Especially useful for new Java versions
+
+### For versions of Java that already have gradle support 
+
+To just build the WAR file locally (in build/libs), change to the `java17` directory under the root of this project and:
+
+```
+./gradlew build
```

Comment:
If wrapper is not checked into source.  Users will need to do `gradle init` to install the wrapper.  Hence, previous review comments.

Reply:
Thanks for catching that!